PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+### Description of work
+
+*Add your own description here. The aim is provide information to help the reviewer review the PR.*
+
+### Issue
+
+*Link to Issue, if there is one.*
```

Comment:
Could put `Closes #XX` here, as some people forget or are unaware of the auto-closing feature.